geom.js

```diff
@@ -22,11 +67,29 @@ class LineSegment {
     this.x2 = x2;
     this.y2 = y2;
   }
+  // Pythagorean theorem = a^{2}+b^{2}=c^{2}
+  length() {
+     if (Math.pow(this.x1, 2) + Math.pow(this.y1, 2)) {
```

Comment:
for `length()` use the distance formula: `return Math.sqrt( Math.pow(this.x2 - this.x1, 2) + Math.pow(this.y2 - this.y1, 2) )`